robot/web/stream/state/state.go

```diff
@@ -452,7 +456,8 @@ func (ss *StreamState) streamH264Passthrough(ctx context.Context) error {
 }
 
 func (ss *StreamState) unsubscribeH264Passthrough(ctx context.Context, id rtppassthrough.SubscriptionID) error {
-	cam, err := camera.FromRobot(ss.robot, ss.Stream.Name())
```

Comment:
We need to turn the SDPTrackName back into a short name for the `FromRobot` lookup to work if the camera is in a remote part